base-notebook/start-notebook.sh

```diff
@@ -4,19 +4,22 @@
 
 set -e
 
+if [[ -n "${JUPYTERHUB_API_TOKEN}" ]]; then
+    echo "WARNING: use start-singleuser.sh instead of start-notebook.sh to start a server associated with JupyterHub."
+    exec /usr/local/bin/start-singleuser.sh "$@"
+    exit
```

Comment:
I think this `exit` does nothing since the `exec` statement ends execution of this script.

Reply:
```suggestion
```